kernel/device.c

```diff
@@ -84,17 +88,22 @@ const struct device *z_impl_device_get_binding(const char *name)
 	 * with CONFIG_* macros), only cheap pointer comparisons will be
 	 * performed. Reserve string comparisons for a fallback.
 	 */
+	LOG_DBG("device_get_binding(%s)", name);
 	for (dev = __device_start; dev != __device_end; dev++) {
 		if (z_device_ready(dev) && (dev->name == name)) {
+			LOG_DBG("   - found");
```

Comment:
IMHO printing multiple lines looks ugly, also printing the function name is not needed because it is printed automatically by the dbg macro.
Why just have `LOG_DBG("%s found", name);` here.

Reply:
I was imagining it might crash, but your change is OK with me.